src/components/SwitchLocaleLink/index.tsx

```diff
@@ -17,46 +14,37 @@ const Container = styled(TYPE.small)`
   margin-top: 1rem !important;
 `
 
-export function SwitchLocaleLink() {
-  const activeLocale = useActiveLocale()
+const useTargetLocale = (activeLocale: SupportedLocale) => {
```

Comment:
if this werent a hook then target locale would never be null i think and then your hook can always return a value w/ to and onClick

Reply:
`targetLocale` is actually conditionally set based on active locale and default locale so I think `null` is a valid state here.